provision/stack/resources/datadog.yaml

```diff
@@ -0,0 +1,100 @@
+warehouses:
+  - name: datadog_wh
+    properties:
+      warehouse_size: xsmall
+      auto_suspend: 60
+      auto_resume: true
+      initially_suspended: true
+
+roles:
+  # Create a new role intended to monitor Snowflake usage. The name of the role is customizable.
+  - name: datadog
+
+privilegeGrants:
+  # Grant privileges on the SNOWFLAKE database to the new role.
+  - objectType: database
+    objectName: snowflake
+    roleName: datadog
+    privilege:
+      - imported privileges
+  # Grant usage to your default warehouse to the role DATADOG.
+  - objectType: warehouse
+    objectName: datadog_wh
+    roleName: datadog
+    privilege:
+      - usage
+  # Grant usage on the database, schema, and table of the event table.
+  - objectType: database
+    objectName: phdata
+    roleName: datadog
+    privilege:
+      - usage
+  - objectType: schema
+    objectName: phdata.core
+    roleName: datadog
+    privilege:
+      - usage
+  - objectType: table
+    objectName: phdata.core.demo_events
```

Comment:
Created it manually in the demo account

Reply:
It had the default event table `snowflake.telemetry.events`, but to keep it similar to the PE account, I manually created `phdata.core.demo_events`.